github-filter-worker/src/bindings.d.ts

```diff
@@ -1,4 +1,5 @@
 declare global {
+    const emojis: KVNamespace
```

Comment:
Should be uppercase.
```suggestion
    const EMOJIS: KVNamespace
```